api/opds.py

```diff
@@ -499,7 +499,7 @@ def _add_link(l):
                 _add_link(d)
 
         for type, value in Configuration.help_uris(self.library):
-            d = dict(href=setting.value, rel="help")
+            d = dict(href=value, rel="help")
```

Comment:
`setting.value` is the value of the last setting that was considered above, in the previous `for` loop.